dateutil/tz/tz.py

```diff
@@ -1726,13 +1724,13 @@ def resolve_imaginary(dt):
     .. doctest::
 
         >>> from dateutil import tz
-        >>> from datetime import datetime
+        >>> import datetime
         >>> NYC = tz.gettz('America/New_York')
-        >>> print(tz.resolve_imaginary(datetime(2017, 3, 12, 2, 30, tzinfo=NYC)))
+        >>> print(tz.resolve_imaginary(datetime.datetime(2017, 3, 12, 2, 30, tzinfo=NYC)))
         2017-03-12 03:30:00-04:00
 
         >>> KIR = tz.gettz('Pacific/Kiritimati')
-        >>> print(tz.resolve_imaginary(datetime(1995, 1, 1, 12, 30, tzinfo=KIR)))
+        >>> print(tz.resolve_imaginary(datetime.datetime(1995, 1, 1, 12, 30, tzinfo=KIR)))
```

Comment:
I think I would prefer these and their like to be `from datetime import datetime`, but I don't feel strongly about this.

Reply:
Also the correct thing to do is I think `1994, 12, 31` which should resolve to `1995, 1, 1`. This was a mistake in the tzdatabase fixed in 2018e.

Reply:
You can change this to:

```python
>>> APIA = tz.gettz('Pacific/Apia')
>>> print(tz.resolve_imaginary(datetime(2011, 12, 30, 12, tzinfo=APIA))
2011-12-31 12:00:00+14:00
```